packages/gatsby-theme-mdx/src/pages/.eslintrc

```
@@ -0,0 +1,11 @@
+{
+  "overrides": [
+    {
+      "files": "./conf.mdx",
+      "globals": {
+        "Speaker": false,
+        "SpeakerList": false
```

```suggestion
        "Speaker": "readonly",
        "SpeakerList": "readonly"
```

https://eslint.org/docs/user-guide/configuring/language-options#using-configuration-files-1

---

`false` means `readonly` too actually

---

@ChristianMurphy Done!